specification/network/resource-manager/Microsoft.Network/stable/2024-03-01/virtualWan.json

```diff
@@ -2634,6 +2634,270 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/vpnGateways/{gatewayName}/vpnConnections/{connectionName}/vpnLinkConnections/{linkConnectionName}/sharedKeys": {
+      "get": {
+        "operationId": "VpnLinkConnections_GetAllSharedKeys",
+        "description": "Lists all shared keys of VpnLink connection specified.",
```

Comment:
Make this a part of a new API version

Reply:
@rkmanda , 2024-03-01 is the latest API version for Microsoft.Network. Can we use 2024-03-01 for our APIs?

Reply:
@rkmanda, I understand your concern about breaking change on this version (2024-03-01). The manifest rollout may already be completed, allowing users to use this version in production. Unfortunately, we cannot control the swagger release process, since NRP team is driving the release. But, this issue does not seem to be specific to our PR. Technically, any change in the 2024-03-01 Swagger release PR might be considered a breaking change, right? Also, our new APIs do not have public documentation yet, so users are not aware of them. Can you please help us work out a plan? I am happy to jump on a call. Thanks!